server/ethereum/index.js

```diff
@@ -17,6 +17,8 @@ const handlePinHashEvent = async (err, event) => {
   try {
     const result = await ipfs.getAndPin(event.returnValues.cid)
```

Comment:
This no longer has to happen, since the pinning occurs upon the HTTP endpoint

Reply:
is that on this PR?? I didn't touch this